src/main/java/Spock.java

```diff
@@ -0,0 +1,62 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class Spock {
+    public static void main(String[] args) {
+        System.out.println("____________________________________________________________");
+        System.out.println("Hello! I'm Spock");
+        Scanner myObj = new Scanner(System.in);
+        System.out.println("What can I do for you?");
+        System.out.println("____________________________________________________________");
+
+        String input = myObj.nextLine();
+        List<Task> list = new ArrayList<>();
+        Task reply = new Task(input, false, 1);
+
+
+
+        for(int i = 1; !input.equals("bye"); input = myObj.nextLine()) {
+
+            System.out.println("____________________________________________________________");
+
+            if (input.equals("list")) {
+                System.out.println("Here are the tasks in your list:");
+                for (int k = 0; k < i-1; k++) {
+                    System.out.println(list.get(k).index + ".[" + list.get(k).getStatusIcon() + "] " + list.get(k).description);
+                }
+            }
+            else if (input.startsWith("mark")) { //TODO: disallow marking a task that is already marked and task_number outside of range
+                int task_number = Integer.parseInt(input.replace("mark ", "")) - 1;
```

Comment:
I think variable names should be in camelCase.
```suggestion
                int taskNumber = Integer.parseInt(input.replace("mark ", "")) - 1;
```

Reply:
Yes, I believe variable names should be in camelCase